src/Hng.Web/Controllers/ProductController.cs

```diff
@@ -0,0 +1,19 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Hng.Web.Controllers
+{
```

Comment:
There's no need for a controller if there's no action

Reply:
Was just setting up a controller for my other teammates towork in